File: backend/src/main/java/com/NBE3_4_2_Team4/global/security/oauth2/disconectService/OAuth2DisconnectService.java
package com.NBE3_4_2_Team4.global.security.oauth2.disconect.service;

import com.NBE3_4_2_Team4.domain.member.member.entity.Member;

public interface OAuth2DisconnectService {
    Member.OAuth2Provider getProvider();
    boolean disconnect(String refreshToken);
}

File: backend/src/main/java/com/NBE3_4_2_Team4/global/security/oauth2/disconnectService/OAuth2DisconnectService.kt
package com.NBE3_4_2_Team4.global.security.oauth2.disconnectService

import com.NBE3_4_2_Team4.domain.member.member.entity.Member

interface OAuth2DisconnectService {
    fun getProvider(): Member.OAuth2Provider?
    fun disconnectSuccess(refreshToken: String?): Boolean
}